vault/vault-mantl/spec.yml

```diff
@@ -15,13 +15,13 @@ depends:
 
 targets:
   - src: "{{.SpecRoot}}/vault-unseal.sh"
-    dest: /usr/local/bin
+    dest: /usr/local/bin/vault-unseal.sh
```

Comment:
I got a weird error where the package wouldn't build unless this was set explicitly.